localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```diff
@@ -69,47 +69,47 @@
  
  innovation_silithid:0 "Silithid"
  innovation_silithid_desc:0 "Silithids are insectoid creatures that are very strong and sophisticated. The chitin plates of their bone armor are intertwined in such a way that it makes it incredibly hard. It seems that the Silithids' intention is to devour everything that stands in their way and that does not belong to their own ecology. This is the only thing they focus on, and this is why they pose a great threat to all life on Azeroth."
 
  innovation_snow_hunter:0 "Snow Hunters"
- innovation_snow_hunter_desc:0 ""
+ innovation_snow_hunter_desc:0 "Our kind has had to bend the elements to our will in order to survive.  The best of our warriors are the hunters who keep us safe."
     
  innovation_thornguard:0 "Thornguards"
  innovation_thornguard_desc:0 "The Thornguards are the finest warriors from our tribe, who defend us from any outsiders.  They are heavily armed and armored; they will stop any who dare to trespass or any who dare to not submit."
 
  innovation_tidehunter:0 "Tidehunters"
- innovation_tidehunter_desc:0 ""
+ innovation_tidehunter_desc:0 "mrglmrglmrglmrgl"
  
  innovation_transcendent:0 "Transcendent Avatars"
  innovation_transcendent_desc:0 "Pandaren revere transcendents as embodiments of wisdom: the spirit of the universe made flesh, residing in the shell of a pandaren host. These are the pandaren’s ultimate heroes: Spiritual avatars capable of summoning vast storms of lighting, smashing mountains with their terrible fists, and breathing winds into existence."
     
  innovation_ursa:0 "Ursas"
- innovation_ursa_desc:0 ""
+ innovation_ursa_desc:0 "Ursas are the biggest, meanest and most feral of furbolgs. They tower above humans and orcs, and can snap elven necks with single paw swipes. They are immensely strong and tough, and focus on improving their natural, bearlike natures. They disembowel dragons with their claws and crush small creatures that come within their grasp."
 
  innovation_war_dinosaur:0 "War Dinsosaurs"
- innovation_war_dinosaur_desc:0 ""
+ innovation_war_dinosaur_desc:0 "Once a dominant species throughout the planet, they have become an endangered remnant of an ancient world that prove to be a force to be reckoned with. Dangerous as well as majestic are the great beasts. Our tamers have successfully learned how to communicate and tame these grand creatures."
 
  innovation_wolfrider:0 "Wolfriders"
- innovation_wolfrider_desc:0 ""
+ innovation_wolfrider_desc:0 "These savage warriors used the speed and ferocity of their mounts to make them a deadly enemy. Faster than those who attacked on foot, they had perfected the tactics of tearing through columns of enemy soldiers like a hot wind."
  
  ######################
  # Early Medieval Era #
  ######################
   
  innovation_bearpaws:0 "Bearpaws"
- innovation_bearpaws_desc:0 ""
+ innovation_bearpaws_desc:0 "Strong and skilled fighters, bearpaws though lightly armed serve a big punch if provoked."
```

Comment:
"Though they are only lightly armed, the bearpaws serve a big punch if provoked as strong and skilled fighters that they are."